Dockerfile

```
@@ -0,0 +1,18 @@
+FROM python:3.7
```

Updating this to be consistent with the version of python tested
```suggestion
FROM python:3.11
```